tethysext/atcore/tests/integrated_tests/models/files_database/file_collection_mixin.py

```diff
@@ -0,0 +1,291 @@
+import datetime
```

Comment:
To be consistent we should rename this to `file_collection_mixin_tests.py`

Reply:
Fixed